pytest_catchlog/plugin.py

```diff
@@ -50,12 +56,60 @@ def pytest_addoption(parser):
         dest='log_date_format', default=DEFAULT_LOG_DATE_FORMAT,
         help='log date format as used by the logging module.'
     )
+    add_option_ini(
```

Comment:
It's not clear to me why there are two different options that appear to configure the same thing. Can you enlighten me here, please?